ui/app/AppLayouts/Wallet/ReceiveModal.qml

```diff
@@ -57,7 +57,7 @@ ModalPopup {
     }
 
     Input {
-	id: txtWalletAddress
+	      id: txtWalletAddress
```

Comment:
This looks weirdly formatted here, but it actually is weirdly formatted right now in the file